### ISSUE TITLE
When creating a child user story from an epic the area of the epic should be inherited by the user story

### ISSUE THREAD

**Issue by rootAvish** [openshiftio/openshift.io#3964]
There might be scenarios prevalent where the user story is of a different work area than it's parent, but I'm sure those situations are far and in between so it's a safe bet to assume it's from the same area and give the user story the area of it's parent.

It's a major annoyance to edit each user story separately and add the correct area type.

**Comment by joshuawilson** [comment]
Related to #3966

**Comment by joshuawilson** [comment]
@openshiftio/uxd-team please advise

**Comment by sbose78** [comment]
Sounds like a fair request. However, we can't set the parent item's area as a default without giving the user an option to opt out of the default. This is because it is not rare to have epics which have stories belonging to different areas.

**Comment by rootAvish** [comment]
> However, we can't set the parent item's area as a default without giving the user an option to opt out of the default

Right, so the way this can then be done is to maybe have a checkbox on the creation form somewhere:

> - [x] Set area of story same as area of parent

Which the user can uncheck if there's a particular user story that is in fact for a different area.

**Comment by serenamarie125** [comment]
Based on a conversation held on 2019-Jan-04 including Dev, PM and UX, the UX team will not be providing support for planner items. Closing this issue since this was solely around gathering requirements.
